src/impls.rs

```diff
@@ -241,3 +233,8 @@ where
         r#ref: TypeExpr::Ref(&T::INFO),
     });
 }
+
+#[cfg(feature = "json_value")]
+impl_native!(serde_json::Value, "unknown");
+#[cfg(feature = "json_value")]
+impl_map!(serde_json::Map);
```

Comment:
For completeness I think `serde_json::Number` should be included as well.
```suggestion
#[cfg(feature = "json_value")]
impl_native!(serde_json::Value, "unknown");
#[cfg(feature = "json_value")]
impl_map!(serde_json::Map);
#[cfg(feature = "json_value")]
impl_native!(serde_json::Number, "number");
```